mrc-ide/hint
# Mrc 2917 Display warnings on review-inputs

## Description

This PR displays warning on `review inputs` step. Although, warnings are displayed on `review inputs` step but data was fetched from genericCharts data and not for other charts. @r-ash has confirmed that it's okay to display warnings generated from genericChart for `review inputs` step, ignoring other charts, so we do not need to worry about none GenericCharts.

Note that warnings are not displayed by default when a user reaches reviewInput step but when time-series is triggered. Science team asked for this behaviour as they had like to review warnings while chart is displayed. 

TO TEST
- load data
- navigate to step 2 (review inputs step)
- click on time-series chart
- see warning displayed


Please don't merge this branch to master yet. 
TO:DO 👍 
Change base hintr: mrc-2918 to master before merging

## Type of version change
_Delete as appropriate_

Minor

## Checklist
